kube/pod.go

```diff
@@ -370,6 +370,15 @@ func getEnvVarsFromConfigs(configs model.ConfigurationVariableSlice, settings Ex
 			continue
 		}
 
+		if config.Name == "HELM_IS_INSTALL" {
+			value := "true"
+			if settings.CreateHelmChart {
+				value = "{{ .Release.IsInstall | quote }}"
```

Comment:
Can this be unset if false? So that `test -n "${HELM_IS_INSTALL}"` would do the right thing.

Reply:
It will be either "true" or "false", which works fine if you want to pass it as a `flag.Bool` to a Go binary (which I do). Using an empty string would work badly, so I would prefer to keep it as it is.